enos/modules/docker_worker/main.tf

```diff
@@ -115,6 +121,14 @@ resource "docker_container" "worker" {
   }
 }
 
+resource "enos_local_exec" "check_address" {
+  depends_on = [
+    docker_container.worker
+  ]
+
+  inline = ["timeout 10s bash -c 'until echo $(curl -s -i \"http://0.0.0.0:${local.port_ops}/health?worker_info=1\") | grep -i \\\"upstream_connection_state\\\":\\\"READY\\\"; do sleep 2; done'"]
```

Comment:
🤢

Reply:
Is there a better way to do this?

Reply:
Haha no it's fine, it's just the escapes within escapes 😂

Reply:
Yea... I couldn't figure out another option